src/main/java/htsjdk/samtools/CRAMIterator.java

```diff
@@ -106,7 +106,7 @@ public CRAMIterator(final SeekableStream seekableStream, final CRAMReferenceSour
         this.containerIterator = containerIterator;
 
         firstContainerOffset = containerIterator.getFirstContainerOffset();
-        records = new ArrayList<SAMRecord>(10000);
+        records = new ArrayList<SAMRecord>();
```

Comment:
How many records are expected in each container? This could have some performance effect, but I understand that there is no explanation for the magic number. Should it be extracted as an default constant at the class level or maybe to an argument to the iterator?

Reply:
@magicDGS hello, thanks for your review, I've extracted it as constant at the class level